### PR TITLE
Alterations to make jerrycan quality update

### PR DESCRIPTION
**Pull request by cammygames**
This allows the jerry can items quality to update when it is used to refuel a vehicle.
Requires changes to qb-weapons and qb-inventory

https://github.com/qbcore-framework/qb-weapons/pull/33
https://github.com/qbcore-framework/qb-inventory/pull/90

### CODE REVIEW

File: source/fuel_client.lua [hunk 1]
```diff
@@ -214,32 +242,33 @@ Citizen.CreateThread(function()
...
-
-										SetPedAmmo(ped, 883325847, 4500)
+						DrawText3Ds(stringCoords.x, stringCoords.y, stringCoords.z + 1.2, Config.Strings.PurchaseJerryCan)
+						if IsControlJustReleased(0, 38) then
+							QBCore.Functions.TriggerCallback('fuel:server:hasJerryCans', function(hasCans, items)  
+								if hasCans then
+									if #items > 0 then
+										local can = items[1]
+					
+										local refillCost = Round(Config.RefillCost * (1 - can.info.ammo / 4500))
```
Comment (duonglt305) [inline source/fuel_client.lua]: You should be change this line to:
local refillCost = Round(Config.RefillCost * (1 - can.info.quality/ 4500))

Comment (cammygames) [reply source/fuel_client.lua]: Thanks! Good spot!

**Comment by xenfovn** [comment]
hi, this works great, but you won't be able to refuel the gas can. it asks you to buy a new gas station, check this out

**Comment by evanillaa** [comment]
greate work thank you

**Review by ARSSANTO — APPROVED** [review]

**Comment by xenfovn** [comment]
fix it pls

**Pull request closed without merging**